rand_isaac/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.1.2] - 2019-06-06
+- Bump `rand_core` version
+- Remove deprecated code
+
```

Comment:
Shouldn't this (and the other `rand_isaac` changes) be in a different commit?

Reply:
I guess so, but does it actually matter?

Reply:
Not really. Maybe just change the commit message to `Prepare rand_jitter 0.2.0 and rand_isaac 0.1.2 release`?